docs/docusaurus.config.js

```diff
@@ -153,6 +153,11 @@ const config = {
 						position: 'left',
 						target: '_self',
 					},
+					{
+						href: './storybook',
+						label: 'Storybook',
```

Comment:
```suggestion
						label: 'Composants graphiques',
```

ou 
```suggestion
						label: 'Design System',
```
ou autre chose ?